lib/protocol/screenshot.js

```diff
@@ -10,8 +10,12 @@
  *
  */
 
-let screenshot = function () {
-    return this.requestHandler.create('/session/:sessionId/screenshot')
+let screenshot = function (opts) {
```

Comment:
Yeah, sorry it wasn't a good solution.

I've changed the code, can you look again please?